### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by Ilanak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,50 @@
...
+
+Installation
+
+$ gem install fluent-plugin-azureactivitylog
+
+## Configuration
+
+```config
+<source>
+  @type azureactivitylog
```
Comment (torosent) [inline README.md]: Add service principal link

File: fluent-plugin-azureactivitylog.gemspec [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+
+Gem::Specification.new do |gem|
+  gem.name          = "fluent-plugin-azureactivitylog"
+  gem.version       = "0.0.1"
+  gem.authors       = ["Ilana Kantorov"]
+  gem.email         = ["ilanak@microsoft.com"]
+  gem.description   = %q{Input plugin for Azure Activity logs.}
+  gem.homepage      = "https://github.com/..."
```
Comment (torosent) [inline fluent-plugin-azureactivitylog.gemspec]: put the real github link

File: lib/fluent/plugin/in_azureactivitylog.rb [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+    super
+    @watcher.terminate
+    @watcher.join
+  end
+
+  private
+
+  def watch
+    while true
+        log.debug "azure activitylog: watch thread starting"
```
Comment (torosent) [inline lib/fluent/plugin/in_azureactivitylog.rb]: check general debug setting

File: lib/fluent/plugin/in_azureactivitylog.rb [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+    # Set Headers
+    request_headers['x-ms-client-request-id'] = SecureRandom.uuid
+    request_headers['accept-language'] = @client.accept_language unless @client.accept_language.nil?
+    path_template = '/subscriptions/{subscriptionId}/providers/microsoft.insights/eventtypes/management/values'
+
+    options = {
+        middlewares: [[MsRest::RetryPolicyMiddleware, times: 3, retry: 0.02], [:cookie_jar]],
+        path_params: {'subscriptionId' => @client.subscription_id},
+        query_params: {'api-version' => api_version, '$filter' => filter, '$select' => @select_filter},
```
Comment (torosent) [inline lib/fluent/plugin/in_azureactivitylog.rb]: add azure link for select params

File: lib/fluent/plugin/in_azureactivitylog.rb (hunk 3 above)
Comment (Ilanak) [reply lib/fluent/plugin/in_azureactivitylog.rb]: I checked it.
Debug messages will be shown only when -vv flag is added to the fluent command.

**Pull request merged**